pages/common/openssl-genpkey.md

```diff
@@ -0,0 +1,16 @@
+# openssl genpkey
+
+> OpenSSL command to generate asymmetric key pairs.
+> More information: <https://www.openssl.org/docs/man1.0.2/man1/genpkey.html>.
+
+- Generate a 2048bit RSA private key and save it to a file:
+
+`openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:2048 -out {{filename.key}}`
```

Comment:
```suggestion
`openssl genpkey -algorithm {{rsa}} -pkeyopt {{rsa_keygen_bits}}:{{2048}} -out {{filename.key}}`
```

Reply:
`rsa`and `rsa_keygen_bits` are not really optional here, since the command is specifically for creating an RSA key. I would just tokenize the bitlength ot `{{2048}}`

Reply:
I'm not talking about optional arguments, I mean an option that specific argument `-algorithm` because other values are allowed.

Reply:
Not really. If you want to generate an RSA key, and that's what this command is about, than `rsa` is the only viable option here. It would be a completely different command otherwise.
Due to the different set of option, I consider it helpful to have separate commands for different key types here.

Reply:
Looks the `tldr` page: the `Show the tar tldr page for Linux` example. Both, `linux` and `tar` are between `{{}}` but if you want the page for `tar` in `linux` this example is the only way to get it. but `linux` and `tar` are argument options. like `rsa` here.

Reply:
OK, if that's the way you want it I'll accept that.

Reply:
Is using one token for the option name/value pair OK?

Reply:
Hrm, I see what's happened here - looks like a case of crossed wires. What about this?
```suggestion
`openssl genpkey -algorithm rsa -pkeyopt rsa_keygen_bits:{{2048}} -out {{filename.key}}`
```

@navarroaxel: Looks like this example is specifically about rsa, so other values don't make sense here because fo the extra `rsa_keygen_bits` argument.